networking/external_dns_operator/nw-installing-external-dns-operator.adoc

```diff
@@ -0,0 +1,30 @@
+[id="nw-installing-external-dns-operator_{context}"]
+= Installing the external dns operator
+
+You can install the External DNS Operator from the web console.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* Prepare external dns operator namespace
+* `Oc create ns external-dns-operator`
```

Comment:
Since this installation will be made from `OperatorHub` console, this namespace can be created automatically. It will be proposed as the default namespace of where the operator should be installed.

Reply:
oh okay. will remove it

Reply:
Done